### PR TITLE
CAL-441 Update documentation to explain how attributes are added to metacards.

### PR DESCRIPTION
**Pull request by bakejeyner**
#### What does this PR do?
This PR updates the documentation to explain how attributes are added to metacards during ingestion/querying.

#### Who is reviewing it? 
@blen-desta
@brjeter
@emanns95
@SmithJosh

#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)
@clockard
@coyotesqrl
@ricklarsen

#### How should this be tested?
1. Build DDF
2. Check to make sure the documentation includes this section

#### What are the relevant tickets?
[CAL-441](https://codice.atlassian.net/browse/CAL-441)
[DDF-3876](https://codice.atlassian.net/browse/3876)

#### What are the relevant PRs?
[DDF](https://github.com/codice/ddf/pull/3311)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by bakejeyner** [comment]
image
image

**Comment by bakejeyner** [comment]
image

File: distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+:title: Attributes Added by Pre-Ingest Plugins
+:type: dataManagement
+:status: published
+:parent: Automatically Added Metacard Attributes
+:summary: How pre-ingest plugins add attributes to metacards.
+:order: 051
+
+* The <<_default_security_attribute_plugin,Default Security Attribute Plugin>> is a configurable plugin that maps system high user attribute values to metacard attribute values.
+"System high user attributes" refers to the attributes on the ${branding}'s https://en.wikipedia.org/wiki/Second-level_domain[Second-Level Domain] user, who is defined in the `etc/users.attributes` file.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc]: ❓ This isn't quite correct. On installation, the user is in the `users.attributes` file but in a hardened system, that file is removed and an external attribute store should be queried. I think we should indicate that here.

@stustison, what is your thought on that?

Comment (bakejeyner) [reply distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc]: Without looking into it more, I think the only store that we pull the "System High Attributes" from is that text file. I'm not sure what happens when it is deleted.

Comment (brjeter) [reply distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc]: I agree with Richard about the hardened system. On doing a few minutes of investigation, it seems the Default Security Attributes Plugin pulls "system high user attributes" from the `UsersAttributesFileClaimsHandler` which requires the `users.attributes` file. Is that really the case?

Comment (brjeter) [reply distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc]: So after more investigation, I figured out that the `users.attributes` file never gets removed. It's the `users.properties` file that gets removed. `users.attributes` does not fully define the FQDN user. It just adds in attributes for that user. Which is also what's used for the "System high user attributes". That should be clarified here.

Comment (bakejeyner) [reply distribution/docs/src/main/resources/content/_managing/attributes-added-by-pre-ingest-plugins.adoc]: Thanks for looking into this! I'll make sure to note that the users.attributes files just defines the attributes and not the entire user.

**Comment by bakejeyner** [comment]
build now

**Pull request merged**